app/src/main/java/net/squanchy/support/lang/Ids.java

```diff
@@ -0,0 +1,21 @@
+package net.squanchy.support.lang;
+
+
+public final class Ids {
+
+    public static long safelyConvertIdToLong(String id) {
+        try {
+            return Long.parseLong(id);
+        } catch (NumberFormatException e) {
+            return -1;
```

Comment:
❌  NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOPE

The IDs are all strings so we don't need to do this in the first place. That said, swallowing errors and propagating invalid data is not acceptable.